chia/wallet/wallet_state_manager.py

```diff
@@ -641,68 +644,82 @@ async def unconfirmed_removals_for_wallet(self, wallet_id: int) -> Dict[bytes32,
 
     async def determine_coin_type(
         self, peer: WSChiaConnection, coin_state: CoinState, fork_height: Optional[uint32]
-    ) -> Optional[WalletIdentifier]:
+    ) -> Tuple[Optional[WalletIdentifier], Optional[Streamable]]:
```

Comment:
Let's not introduce `Tuple` returns.  Make a dataclass so the elements have names.